### ISSUE TITLE
Modification in Nodes - error creating EKS Cluster

### ISSUE THREAD

**Issue by mitesh51** [hashicorp/learn-terraform-provision-eks-cluster#19]
If I want to change instance type or add new worker Group then I get: Error: 

Error: error creating EKS Cluster (terraform-eks-dev-i8DeuS8U): ResourceInUseException: Cluster already exists with name: terraform-eks-dev-i8DeuS8U
{
  RespMetadata: {
    StatusCode: 409,
    RequestID: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
  },
  ClusterName: "terraform-eks-dev-i8DeuS8U",
  Message_: "Cluster already exists with name: terraform-eks-dev-i8DeuS8U"
}

**Comment by mitesh51** [comment]
1) I created an EKS cluster using same code and it worked fine. I am using Windows machine to perform all operations so I added following script:
  wait_for_cluster_interpreter = ["C:/Program Files/Git/bin/sh.exe", "-c"]
  wait_for_cluster_cmd         = "until curl -sk $ENDPOINT >/dev/null; do sleep 4; done"
2) In worker Groups, I changed instance type and executed terraform apply in the first block and I got the error.  
3) I added another worker group block and executed terraform apply and got same error.
